config/routes.rb

```diff
@@ -177,6 +177,8 @@
       resources :watched_words, only: [:index, :create, :update, :destroy] do
         collection do
           get "action/:id" => "watched_words#index"
+          get "action/:id/download" => "watched_words#download"
+          delete "action/:id/clear_all" => "watched_words#clear_all"
```

Comment:
You don't need to create a new endpoint for clearing out all the watched words.

Using the `DELETE` verb on the "collection" is enough.

```suggestion
          delete "action/:id" => "watched_words#clear_all"
```